src/styles/blocks/navigation.scss

```diff
@@ -0,0 +1,44 @@
+.nav {
```

Comment:
8-th point of the CHECKLIST 👉 [checklist.md](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md)